identity-apps-core/apps/recovery-portal/src/main/resources/org/wso2/carbon/identity/mgt/recovery/endpoint/i18n/Resources.properties

```diff
@@ -37,15 +37,29 @@ sign.up.heading=Sign Up
 sign.up.button=Sign Up
 # <!-- End of Sign Up Screen Translations -->
 
+# <!-- SMS OTP -->
+sms.otp.heading=OTP Verification
+enter.code.sent.smsotp=Enter the code sent to your mobile phone
+sms.otp.resend.code=Resend Code
+sms.otp.submit.button=Continue
```

Comment:
Don't we provide custom text configuration capability for these strings?

Reply:
This is already added for branding through the authentication portal for the same UI. Added the branding comment for reference with https://github.com/wso2/identity-apps/pull/6418/commits/f078ab464fe1a4f35d2b31c679acec6264746099